ts/features/fims/screens/FimsWebviewScreen.tsx

```diff
@@ -68,18 +70,36 @@ const FimsWebviewScreen = () => {
       );
       return;
     }
-    const url = new URLParse(maybeParams.value.url as string, true);
+
+    const maybeParsedUrl = maybeFimsDomain.chain(domain => {
+      const parsed = new URLParse(domain as string, true);
+      return parsed.protocol && parsed.protocol === "https"
```

Comment:
```suggestion
      return parsed.protocol && parsed.protocol === "https:"
```
The `protocol` has the `:` at the end of the string, in this way it will work fine! 🙏